### PR TITLE
Adding WebNLG dataset (GEM benchmark)

### PR DESCRIPTION
**Pull request by jordiclive**
#18 
Decoding takes 1.2 seconds per example (without batching) on RTX Titan machine.

### CODE REVIEW

**Comment by jordiclive** [comment]
Can someone fix this file that has already been integrated: [example_bias_shades_prompt_generation_code.py](https://github.com/bigscience-workshop/evaluation/blob/main/social-impact-group/BiasShades/example_bias_shades_prompt_generation_code.py)

**Comment by wilsonyhlee** [comment]
> Can someone fix this file that has already been integrated: [example_bias_shades_prompt_generation_code.py](https://github.com/bigscience-workshop/evaluation/blob/main/social-impact-group/BiasShades/example_bias_shades_prompt_generation_code.py)

@jordiclive Sure thing. I will fix it today.

File: requirements.txt [hunk 1]
```diff
@@ -4,3 +4,4 @@ tensorflow==2.5.0
 torch==1.9.0
 tqdm==4.62.0
 transformers==4.9.1
+sacrebleu
```
Comment (wilsonyhlee) [inline requirements.txt:7]: Can you please specify a fixed version for this package?

**Review by wilsonyhlee — CHANGES_REQUESTED** [review]
Looks great overall. Thank you for contributing this dataset. I just left one small comment. 

Also, the style errors on he main branch have been fixed. Please feel free to rebase off of it.

File: requirements.txt (hunk 1 above)
Comment (jordiclive) [reply requirements.txt]: specified :)

**Comment by jordiclive** [comment]
thanks!

**Review by wilsonyhlee — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**